Added GitKraken, Mailspring, Atom, Zsh and Oh My Zsh

karanrajpal14 opened the pull request.

cmoscardi committed:
packages submodule

cmoscardi committed:
packages update

cmoscardi committed:
fixing mongodb logo

cmoscardi committed:
test harness update

cmoscardi committed:
oops forgot to pass the parameters

cmoscardi committed:
amd64 only for mongo

cmoscardi committed:
can use GET or POST, script.sh is now a symlink

cmoscardi committed:
seed_packages sets a URI based on argument

cmoscardi committed:
test script sticks to convention

cmoscardi committed:
test script passes DB URI

cmoscardi committed:
version string fix

cmoscardi committed:
including vagrant

cmoscardi committed:
correct mimetype

cmoscardi committed:
vagrant package fix

cmoscardi committed:
only 14.04 and 16.04 supported for now

cmoscardi committed:
README update

cmoscardi committed:
fixing database schema for cascading issues

cmoscardi committed:
adding sqlite

cmoscardi committed:
README update

cmoscardi committed:
README update for new repo name

cmoscardi committed:
adding skype, steam packages

cmoscardi committed:
ensure add-apt-repository exists

cmoscardi committed:
reversed order

cmoscardi committed:
readme update for https

cmoscardi committed:
fixing steam?

cmoscardi committed:
7zip

cmoscardi committed:
testing noninteractive

cmoscardi committed:
a more sustainable steam fix?

cmoscardi committed:
steam workaround?

cmoscardi committed:
another responses attempt

cmoscardi committed:
tab characters? pretty please?

cmoscardi committed:
debug

cmoscardi committed:
checking selections

cmoscardi committed:
deb file as per SO question

cmoscardi committed:
wget

cmoscardi committed:
dependency issues? really?

cmoscardi committed:
skype deb from MS

cmoscardi committed:
correct wget options

cmoscardi committed:
omg microsoft

cmoscardi committed:
skype dependencies

cmoscardi committed:
omg forgot to remove skype package

cmoscardi committed:
apt purge always exits 0

cmoscardi committed:
zip/unzip, gzip

cmoscardi committed:
updating 7zip image location

cmoscardi committed:
inkscape, gimp

cmoscardi committed:
ugh

karanrajpal14 committed:
Moved Sublime Text 3 to "Editors" category

karanrajpal14 committed:
Added Visual Studio Code

karanrajpal14 committed:
Added qBittorrent

karanrajpal14 committed:
Added Android Studio

Installing Android Studio via a PPA that contains automatically generated builds for Ubuntu
[source](https://github.com/mfonville/android-studio)

karanrajpal14 committed:
Added GitKraken

karanrajpal14 committed:
Added Mailspring

karanrajpal14 committed:
Added Atom

karanrajpal14 committed:
Added Zsh

karanrajpal14 committed:
Added Oh My Zsh

cmoscardi commented:
Not clear to me what that breakage is - I'm starting by just re-running the build to see if the error was spurious.

karanrajpal14 commented:
I'm not sure what the problem is as well. I went through the logs and I couldn't find anything worthwhile. Something's probably up with Zsh and Oh My Zsh.

cmoscardi commented:
It may actually be something unrelated, as I see something about chrome being in multiple places in `sources.list`? I'll take a closer look.

From the full error log here: https://s3.amazonaws.com/archive.travis-ci.org/jobs/312993881/log.txt?X-Amz-Expires=30&X-Amz-Date=20171207T154934Z&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJRYRXRSVGNKPKO5A/20171207/us-east-1/s3/aws4_request&X-Amz-SignedHeaders=host&X-Amz-Signature=ecd9151cee39731ac6d59112e73d33df4790836e47e3137781070b5b772d80e3

karanrajpal14 commented:
I'm unable to access the log. The link was temporary perhaps? Try to have a look and let me know. Else I'll revert Oh My Zsh and try. I'll download 14.04 and run it on Virtualbox and check it out.

karanrajpal14 committed:
Ensuring git is pre-installed

The installation fails on a fresh Ubuntu 16.04 install as git is not pre installed

karanrajpal14 committed:
Logging script.sh contents

karanrajpal14 commented:
I'm not sure about what the issue is @cmoscardi 
Looks like there's some other issue. Is something wrong with the environment perhaps?
I just installed all of these packages on a fresh Ubuntu 16.04 install and they all work.
